tutorial/lesson_07_multi_stage_pipelines.cpp

```diff
@@ -141,15 +143,17 @@ int main(int argc, char **argv) {
 
         // Blur it horizontally:
         Func blur_x("blur_x");
-        blur_x(x, y, c) = (input_16(x-1, y, c) +
+        blur_x(x, y, c) = (input_16(x - 1, y, c) +
```

Comment:
Applying a fussier standard here because these are tutorials. The trailing 4 on a new line is yuck.